mesonbuild/interpreter/interpreterobjects.py

```diff
@@ -1142,3 +1142,13 @@ class StructuredSourcesHolder(ObjectHolder[build.StructuredSources]):
 
     def __init__(self, sources: build.StructuredSources, interp: 'Interpreter'):
         super().__init__(sources, interp)
+
+class OverrideExecutableHolder(BuildTargetHolder[build.OverrideExecutable]):
+    def __init__(self, exe: build.OverrideExecutable, interpreter: 'Interpreter') -> None:
+        super().__init__(exe, interpreter)
+        self.methods.update({'version': self.version_method})
+
+    @noPosargs
+    @noKwargs
+    def version_method(self, args: T.List[TYPE_var], kwargs: TYPE_kwargs) -> str:
```

Comment:
If this only works starting 1.9, it should be annotated with FeatureNew to warn people of that fact.